ports/ctstraffic/portfile.cmake

```
@@ -0,0 +1,29 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO microsoft/ctsTraffic
+    REF 88a415197951912fc70e440b31cad8d4ff4ea68a
+    SHA512 152ee25d1ba70c68c5bae61ee08d1d2905efd28a10c48672de852c8ee9d0964a9202814cdcc40bca712ef69f952630509810827226861dad16516f5e0827d879
+    HEAD_REF master
+)
+

```

```suggestion

set(VCPKG_BUILD_TYPE release) # Windows port only includes tools.

```
Only build the release mode.